code/modules/mob/living/carbon/human/life.dm

```diff
@@ -35,6 +35,8 @@
 		//heart attack stuff
 		handle_heart()
 
+		handle_alcohol() //Alcohol
+

```

Comment:
maybe move it to where dizziness is handled, handle_status_effect() or something